clang/test/dpct/cmake_migration/case_051/ref.txt

```diff
@@ -1 +1 @@
-// CHECK: Warning: CMake build script file like CMakeLists.txt is not found, so no CMake build script file will be migrated.
+// CHECK: Warning: CMake or Python setup build script file like CMakeLists.txt/package.cmake or setup.py is not found, so no CMake/ Python setup build script file will be migrated.
```

Comment:
setup.py is not necessary, take https://github.com/NVlabs/stylegan3/tree/main/torch_utils  for example, the py files are:
```
./persistence.py
./training_stats.py
./ops/bias_act.py
./ops/conv2d_gradfix.py
./ops/conv2d_resample.py
./ops/grid_sample_gradfix.py
./ops/filtered_lrelu.py
./ops/__init__.py
./ops/upfirdn2d.py
./ops/fma.py
./misc.py
./__init__.py
./custom_ops.py

```
So, it is different from CMake scripts, where CMakeLists.tx is necessary, and the warning msg for py script should be updated.

Reply:
Removed "setup" word from the feature